Dockerfile

```diff
@@ -1,9 +1,17 @@
 FROM python:3.8
 
-WORKDIR /app
+RUN useradd -ms /bin/bash app
```

Comment:
Interesting. What is it for?

Reply:
It creates user `app` with default shell `bash` and creates a home directory for him